scene/3d/skeleton_modification_3d_fabrik.cpp

```diff
@@ -619,10 +523,115 @@ void SkeletonModification3DFABRIK::_bind_methods() {
 }
 
 SkeletonModification3DFABRIK::SkeletonModification3DFABRIK() {
-	stack = nullptr;
 	is_setup = false;
 	enabled = true;
 }
 
 SkeletonModification3DFABRIK::~SkeletonModification3DFABRIK() {
 }
+
+void SkeletonModification3DFABRIK::_notification(int p_what) {
+	switch (p_what) {
```

Comment:
- set_process_internal / physics_process_internal -> superclass
- remove _cache functions everywhere
- get_node should be done in NOTIFICATION_READY
- take physics process and move it to the superclass
- set_execution_mode() should update the set_physics_process
- set_skeleton() should also get_node again
- delete is_setup and all related checks
- enabled -> set_process_internal(false) 

This is direct all like 10-15 modifications.